packages/testkit/package.json

```diff
@@ -6,6 +6,16 @@
     "TESTING",
     "E2E"
   ],
+  "module": "dist/esm/index.js",
+  "types": "dist/index.d.ts",
+  "exports": {
+    ".": {
+      "require": "./dist/index.js",
+      "import": "./dist/esm/index.js",
+      "types": "./dist/index.d.ts"
+    },
+    "./package.json": "./package.json"
```

Comment:
this would create a @vibe/testkit/package.json entry
is this really needed?

Reply:
if you just want to export the package.json, it gets exported by default anyway, so no need to declare it in `files` or in `exports`

Reply:
see also https://www.npmjs.com/package/@vibe/testkit?activeTab=code

Reply:
removed